scipy/interpolate/tests/test_polyint.py

```diff
@@ -408,5 +417,106 @@ def test_all_zeros(self):
         assert_equal(pch(xx), 0.)
 
 
+class TestCubicSpline(object):
```

Comment:
Should this be derived from `TestCase` instead of `object`? I'm asking only because I've usually seen it written the other way.